### PR TITLE
docs: clarify ip ranges and add cidr reservation

### PR DESCRIPTION
**Pull request by scottd018**
This PR clarifies the following:

- Egress IP selection as related to the VPC range
- Adds recommendation for reserving the IPs via a CIDR reservation

### CODE REVIEW

File: content/docs/rosa/egress-ip/_index.md [hunk 1]
```diff
@@ -83,13 +83,27 @@ oc get node -o json | \
...
 
-### Create the Egress IP Rule
+### Create the Egress IP Rule(s)
 
 > **NOTE:** generally speaking it would be ideal to [label the nodes](#label-the-nodes) prior to assigning 
 the egress IP addresses, however there is a bug that exists which needs to 
 be fixed first.  Once this is fixed, the process and documentation will
 be re-ordered to address this.  See https://issues.redhat.com/browse/OCPBUGS-4969
 
+#### Identity the Egress IPs
```
Comment (Conan-Kudo) [inline content/docs/rosa/egress-ip/_index.md:93]: `Identity` -> `Identify`

File: content/docs/rosa/egress-ip/_index.md [hunk 2]
```diff
@@ -83,13 +83,27 @@ oc get node -o json | \
...
+that the egress IPs that you select should exist as a part of the subnets in which the worker 
+nodes are provisioned into.
+
+#### Reserve the Egress IPs
+
+It is recommended, but not required, to reserve the egress IPs that you have requested to avoid 
+conflicts with the AWS VPC DHCP service. To do so, you can request 
+explicit IP reservations by following the following AWS doc:
+
+https://docs.aws.amazon.com/vpc/latest/userguide/subnet-cidr-reservation.html
```
Comment (Conan-Kudo) [inline content/docs/rosa/egress-ip/_index.md:105]: This should be turned into a link.

**Review by Conan-Kudo — CHANGES_REQUESTED** [review]
Looks good, just some minor fixes needed!

File: content/docs/rosa/egress-ip/_index.md (hunk 1 above)
Comment (scottd018) [reply content/docs/rosa/egress-ip/_index.md]: Fixed in 89ee7161ef1adb3067c25eaec742ebd089806719

**Review by Conan-Kudo — APPROVED** [review]

**Pull request merged**